examples/ocean-dbtc-burning-bot/src/main.ts

```diff
@@ -0,0 +1,160 @@
+import { WIF } from '@defichain/jellyfish-crypto'
+import { WalletClassic } from '@defichain/jellyfish-wallet-classic'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import { fromAddress } from '@defichain/jellyfish-address'
+import { MainNet } from '@defichain/jellyfish-network'
+import { WhaleWalletAccount } from '@defichain/whale-api-wallet'
+import { CTransactionSegWit, TransactionSegWit } from '@defichain/jellyfish-transaction'
+import { BigNumber } from 'bignumber.js'
+import { AddressToken } from '@defichain/whale-api-client/dist/api/address'
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+const BURN_ADDRESS_SCRIPT = fromAddress('8defichainBurnAddressXXXXXXXdRQkSm', 'mainnet')!.script
+
+/**
+ * Initialize WhaleApiClient connected to ocean.defichain.com/v0
+ */
+const client = new WhaleApiClient({
+  url: 'https://ocean.defichain.com',
+  version: 'v0'
+})
+
+// You can `dumpprivkey [address]` with defi-cli, only Bech32 address (df1...) are supported.
+const PRIVATE_KEY = 'PRIVATE_KEY_IN_WIF'
+
+export async function main (): Promise<void> {
+  const wallet = new WalletClassic(WIF.asEllipticPair(PRIVATE_KEY))
+  const sequencer = new BurnProgram(client, wallet)
+
+  // Refill UTXO if balance is getting low
+  if (await sequencer.accountToUTXO()) {
+    return
+  }
+
```

Comment:
Logic doesn't reach here after utxos balance >= 1.1

Reply:
Yup, it's deliberate to prevent mempool conflict, the burn is sequenced in 3 steps. With random execution interval time of 45 minutes to 2 hours.

Reply:
Oops I actually misread, it only proceed burn/swap if "insufficient" utxos balance.